OpenDBDiff.Schema.SQLServer.Generates/Model/Util/FormatCode.cs

```diff
@@ -6,30 +6,25 @@ namespace OpenDBDiff.Schema.SQLServer.Generates.Model.Util
 {
     internal static class FormatCode
     {
+        private static readonly char[] TrimCharacters = { ' ', '\r', '\n', '\t' };
```

Comment:
So this is slightly different than before: it will trim mixes of '\r', '\n', '\t' like before, but also whitespaces mixed with them. I guess that's inline with the method anyway ?

Reply:
Yes, it's fine.